src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateController.php

```diff
@@ -90,32 +91,47 @@ public function __construct(
      * @param string $inputId
      * @return array|RedirectResponse
      */
-    public function typeAction(Request $request, $serviceId, $targetEnvironment, string $inputId)
+    public function typeAction(Request $request, $serviceId, string $targetEnvironment, string $inputId)
     {
         $service = $this->authorizationService->changeActiveService($serviceId);
 
         $choices = $this->protocolChoiceFactory->buildOptions();
```

Comment:
I guess the choices, and the command declared below are now unused code. Perhaps the command can be rid of entirely? 

:wastebasket: time!

Reply:
't has been incinerated!

Reply:
though the choices are still needed for construction of the form.